ccloud/docs/includes/set-acls-destination.rst

```diff
@@ -0,0 +1,15 @@
+|crep| must have authorization to read Kafka data from the origin cluster and write Kafka data in the destination |ccloud| cluster.
+|crep| should be run with a |ccloud| service account, not super user credentials, so use |ccloud| CLI to configure appropriate ACLs for the service account id corresponding to |crep| in |ccloud|.
+For more details on |crep| ACLs, see :ref:`replicator_security_overview`.
+
+.. sourcecode:: bash
+
+   ccloud kafka acl create --allow --service-account <service-account-id> --operation CREATE --topic <replicated-topic>
+   ccloud kafka acl create --allow --service-account <service-account-id> --operation WRITE --topic <replicated-topic>
+   ccloud kafka acl create --allow --service-account <service-account-id> --operation READ --topic <replicated-topic>
+   ccloud kafka acl create --allow --service-account <service-account-id> --operation DESCRIBE --topic <replicated-topic>
+   ccloud kafka acl create --allow --service-account <service-account-id> --operation DESCRIBE-CONFIGS --topic <replicated-topic>
+   ccloud kafka acl create --allow --service-account <service-account-id> --operation ALTER-CONFIGS --topic <replicated-topic>
+   ccloud kafka acl create --allow --service-account <service-account-id> --operation DESCRIBE --cluster-scope
+   ccloud kafka acl create --allow --service-account <service-account-id> --operation CREATE --cluster-scope
```

Comment:
If you have `CREATE` ACLs on the topic then you shouldn't need `CREATE` ACLs on the cluster, although if they are replicating a lot of topics, it could be easier for them to just give `CREATE` on cluster instead of individually giving `CREATE` ACLs to each topic.

Reply:
As discussed before, we can remove the `CREATE` on cluster level here.